hazelcast-jet-distribution/src/root/release_notes.txt

```diff
@@ -39,14 +39,16 @@ Thank you for your valuable contributions!
 
 2. Enhancements
 
-[extensions] Ensure naming conventions for sources/sinks (#2685)
-
 3. Fixes
 
-[core] Jet now doesn't close System.out during JVM shutdown (#2649)
+[core] Fixed queue size display in `DAG.toDotString()` when default is changed (#2887).
 
 4. Breaking Changes
 
+[core] The `DAG.toDotString(int defaultParallelism)` method signature is now
+ `DAG.toDotString(int defaultLocalParallelism, int defaultQueueSize)`. Callers must
+ now supply the queue size that will be shown if not overriden on the edge.
```

Comment:
Same typo as other

Reply:
I simplified now this note to:
```
[core] Added `defaultQueueSize` parameter to `DAG.toDotString(int)` method.
```
Similar to other existing release notes.